auspice/server/getDatasetHelpers.js

```diff
@@ -36,9 +36,26 @@ const splitPrefixIntoParts = (prefix) => {
   /* The first part of the prefix is an optional source name.  The rest is the
    * source path parts.
    */
-  const sourceName = sources.has(prefixParts[0])
-    ? prefixParts.shift()
-    : "core";
+  let sourceName;
+
+  switch (prefixParts[0]) {
+    case "core":
```

Comment:
This makes URLs like `/core/zika` valid, which I don't think we actually want to do.